content/integrate/redis-data-integration/architecture.md

```diff
@@ -68,6 +69,28 @@ RDI automatically enters a second phase called *change streaming*, where
 changes in the data are captured as they happen. Changes are usually
 added to the target within a few seconds after capture.
 
+## At-least-once delivery guarantee
+
+RDI guarantees *at-least-once delivery* to the target. This means that
+a given change will never be lost, but it might be added to the target
+more than once. Apart from a slight performance overhead, adding a
+change multiple times is harmless because the multiple writes
+are [*idempotent*](https://en.wikipedia.org/wiki/Idempotence) (that is
+to say that all writes after the first one make no change to the
+overall state).
```

Comment:
I'd change this sentence to:

"Apart from a slight performance overhead, adding a change multiple times is harmless because all writes after the first one make no change to the overall state."